src/library_html5.js

```diff
@@ -2759,6 +2759,16 @@ var LibraryJSEvents = {
     return !GL.contexts[target] || GL.contexts[target].GLctx.isContextLost(); // No context ~> lost context.
   },
 
+#if USE_WEBGPU
+  // TODO(kainino0x): make it possible to actually create devices through webgpu.h
+  emscripten_webgpu_get_device__deps: ['$WebGPU'],
+  emscripten_webgpu_get_device__postset: 'WebGPU.initManagers();',
+  emscripten_webgpu_get_device: function() {
+    assert(Module['preinitializedWebGPUDevice']);
+    return WebGPU.mgrDevice.create(Module['preinitializedWebGPUDevice']);
```

Comment:
Likely this is too new for closure to be aware of it, so `WebGPU['mgrDevice']['create']` is safer.

Reply:
To test this I tried a build with `--closure 1`, and actually there are a ton of issues where names aren't preserved correctly. Not just here, but also e.g. calls out to WebGPU in the browser. Is the easiest thing to go through and refactor everything with `['...']`?

Reply:
Seems like the only other option for the browser calls is to write a whole closure externs file for them.

Reply:
Fixed!